src/app/context/SettingsContext.tsx

```diff
@@ -107,6 +112,21 @@ export const SettingsProvider = ({
     return getFormattedNumberUtil({ amount, locale: settings.locale });
   };
 
+  const getFormattedInCurrency = (
+    amount: number | string,
+    currency = "BTC" as ACCOUNT_CURRENCIES
```

Comment:
Can this be `currency = ACCOUNT_CURRENCIES['BTC']`?

Reply:
🤔 Don´t hink so, this is not an Enum.